wav.c

```diff
@@ -100,4 +100,23 @@ void wav_finalize_header(FIL * file, unsigned long size) {
 		return;
 		
 	f_write(file, &size, 4, &ret);
-}	
+}
+
+unsigned long wav_header_read_duration(FIL * file) {
+	unsigned int read;
+	
+	f_read(file, read_buffer, sizeof(read_buffer), &read);
+	if(read != sizeof(read_buffer)) 
+		return 0;
+	
+	if(read_buffer[1] - 36 != read_buffer[10])
```

Comment:
Please add comments for these tests, otherwise it is hard to know what the code means.

Reply:
they are sanity check of the wav header file. I can comment but the rest of the code is not... I want not to instore good habit ;-)

Reply:
ok, then write a small comment above them, something like "sanity checks about the integrity of the wave file".